core/commands/patch.go

```diff
@@ -0,0 +1,115 @@
+package commands
+
+import (
+	"fmt"
+	"io"
+	"strings"
+	"time"
+
+	mh "github.com/ipfs/go-ipfs/Godeps/_workspace/src/github.com/jbenet/go-multihash"
+	context "github.com/ipfs/go-ipfs/Godeps/_workspace/src/golang.org/x/net/context"
+
+	cmds "github.com/ipfs/go-ipfs/commands"
+	u "github.com/ipfs/go-ipfs/util"
+)
+
+var PatchCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline:          "Mutate a given merkledag object and return the modified node",
```

Comment:
I'd avoid “mutate” for immutable IPFS objects ;).  Maybe “Create a new Merkle DAG object based on an existing object”?  Or “Create a new Merkle DAG object that's similar to an existing object”?  Or something… ;)